config/config.go

```diff
@@ -154,8 +154,8 @@ type VPCProviderConfig struct {
 	//valid values (gc|g2), if unspecified, GC will take precedence(if both are specified)
 	//during e2e test, user can specify its own preferred type during execution
 	VPCTypeEnabled       string `toml:"vpc_type_enabled" envconfig:"VPC_TYPE_ENABLED"`
-	VPCBlockProviderName string `toml:"vpc_block_provider_name" envconfig:"VPC_BLOCK_PROVIDER_NAME"`
 	VPCBlockProviderType string `toml:"provider_type"`
+	VPCProviderType      string `toml:"vpc_provider_type"`
```

Comment:
I think lets not change the existing one, add new variable which we can use over the time for block as well otherwise it will break the thinks

Reply:
I had made parallel changes to block library but as discuss lets not change anything impacting block